### PR TITLE
Added Entrance Exams for Prestigious Academies, Updated Education Module Documentation

### PR DESCRIPTION
**Pull request by IllianiCBT**
Introduced a system to track and manage failed education applications. Implemented UI updates to show relevant messages and prevent reapplication if previously failed. Additionally, refactored code for clarity and maintainability.

The calculation for entry exams is:

`14 - Faculty Skill` vs `2d6 + (Intelligence / 4)`

Intelligence modifier is not included if personalities are disabled.

Only prestigious academies are affected by entrance exams.

### CODE REVIEW

**Comment by IllianiCBT** [comment]
The application chances for various academies are as follows:

**Ultra-Green** = 83.33%
**Green** = 72.22%
**Regular** = 58.33%
**Veteran** = 41.67%
**Elite** = 27.78%
**Gunslinger Program [3062]** = 16.67%
**Gunslinger Program** = 16.67%
**Special Armed Services Training Center** = 2.78%

File: MekHQ/src/mekhq/campaign/personnel/education/EducationController.java [hunk 1]
```diff
@@ -40,6 +40,86 @@
...
 public class EducationController {
+
+    private static final MMLogger logger = MMLogger.create(EducationController.class);
+    public static final int BASE_TARGET_NUMBER = 7;
+    public static final int DEFAULT_FACULTY_SKILL = 7;
+
+    /**
+     * Checks eligibility for enrollment in an academy.
+     *
+     * @param campaignOptions the campaign options
```
[the remark this answers is not in the document]
Comment (IllianiCBT) [reply MekHQ/src/mekhq/campaign/personnel/education/EducationController.java]: Resolved

**Comment by IllianiCBT** [comment]
Added a campaign option to allow users to adjust the base target number for entrance exams.

Also removed the campaign option that related to CamOps Reputation Divider, as that has been replaced by the CamOps Reputation rewrite - apparently I forgot to remove the option settings from `CampaignOptions.java`

**Review by HammerGS — APPROVED** [review]
Tested and seems to work.

**Pull request merged**